plugins/GSdx/GS.h

```diff
@@ -232,15 +232,13 @@ enum class GSRendererType : int8_t
 {
 	Undefined = -1,
 	DX9_HW,
-	DX9_SW,
-	DX1011_HW = 3,
-	DX1011_SW,
-	Null = 11,
+	DX1011_HW,
```

Comment:
please don't change the value here. If would be painful for me, potentially some value are still hardcoded in the code. And I work only with the ini file.